backend/editor/entries.py

```diff
@@ -253,6 +266,18 @@ def list_existing_projects(self):
         result = get_current_transaction().run(query)
         return result
 
+    def list_all_closed_projects(self):

```

Comment:
As said above, I would prefer that we factor  list_all_open_projects and list_all_closed_projects into a single method. What do you think ?

Reply:
Yep, I think it makes sense to club it into a single method. Using query params, it'll be easier to list projects with a single endpoint as well.